.rubocop_todo.yml

```diff
@@ -971,6 +971,7 @@ Style/PerlBackrefs:
 Style/PredicateName:
   Exclude:
     - 'spec/**/*'
+    - 'src/api/app/controllers/webui/kiwi/images_controller.rb'
```

Comment:
I open a discussion on Rubocop because I don't like this for controllers: https://github.com/bbatsov/rubocop/issues/4479 They have some arguments to enable this, but to be honest I am not convinced that this should be enabled (and I think @mdeniz and @DavidKang also). I would considering excluding the controllers when enabling this cop. :thinking: 

@mschnitzer do you have a good reason to want to have this enable?